pkg/util/log/log.go

```diff
@@ -1,57 +1,123 @@
 package log
 
 import (
+	"fmt"
+
+	"github.com/elastic/beats/v7/libbeat/logp"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	debugSelector = "apic-agents"
+	traceSelector = "apic-agents-trace"
+)
+
+var isLogP bool
+
+//SetIsLogP -
+func SetIsLogP() {
```

Comment:
Could just have a single fn SetIsLogP(bool). Don't know which is better. In fact, in combination with a default value, you probably never need to even have one of the functions. If the default is true, you might only ever need to set it to false. Or if the default is false, you might only ever need to set it to true.

Reply:
I orig just wanted Set, but the tests needed it to be unset.  In actual usage unset is not needed

Reply:
ok